README.md

```diff
@@ -50,16 +51,19 @@ Usage
 
 `cibuildwheel` runs inside a CI service. Supported platforms depend on which service you're using:
 
-|                 | Linux | macOS | Windows | Linux ARM |
-|-----------------|-------|-------|---------|--------------|
-| GitHub Actions  | ✅    | ✅    | ✅      | ✅¹          |
-| Azure Pipelines | ✅    | ✅    | ✅      |              |
-| Travis CI       | ✅    |       | ✅      | ✅           |
-| AppVeyor        | ✅    | ✅    | ✅      |              |
-| CircleCI        | ✅    | ✅    |         |              |
-| Gitlab CI       | ✅    |       |         |              |
+|                 | Linux | macOS | Windows | Linux ARM | macOS ARM |
+|-----------------|-------|-------|---------|-----------|-----------|
+| GitHub Actions  | ✅    | ✅    | ✅      | ✅¹       | ✅²       |
+| Azure Pipelines | ✅    | ✅    | ✅      |           | ✅²       |
+| Travis CI       | ✅    |       | ✅      | ✅        | ✅²       |
+| AppVeyor        | ✅    | ✅    | ✅      |           | ✅²       |
+| CircleCI        | ✅    | ✅    |         |           | ✅²       |
+| Gitlab CI       | ✅    |       |         |           | ✅²       |
+| Cirrus CI       | ✅    | ✅³   | ✅      | ✅        | ✅        |
 
 <sup>¹ [Requires emulation](https://cibuildwheel.readthedocs.io/en/stable/faq/#emulation), distributed separately. Other services may also support Linux ARM through emulation or third-party build hosts, but these are not tested in our CI.</sup><br>
+<sup>² [Uses cross-compilation](https://cibuildwheel.readthedocs.io/en/stable/faq/#universal2). It is not possible to test `arm64` and the `arm64` part of a `universal2` wheel on this CI platform.</sup><br>
+<sup>³ [Uses cross-compilation](https://cibuildwheel.readthedocs.io/en/stable/faq/#universal2). Thanks to Rosetta 2 emulation, it is possible to test `x86_64` and both parts of a `universal2` wheel on this CI platform.</sup><br>
```

Comment:
As I said in my previous comment, I'm not sure cross-compilation is the right term here, could anyone confirm whether "emulation" makes more sense here?
```suggestion
<sup>³ [Uses Rosetta 2 emulation](https://cibuildwheel.readthedocs.io/en/stable/faq/#universal2). Thanks to the emulation, it is possible to test `x86_64` and both parts of a `universal2` wheel on this CI platform.</sup><br>
```

Reply:
Well, technically it's both! Cross-compilation is right, the wheel is built using cross-compilation, but we can still test the wheel using Rosetta 2 emulation.

I think the existing copy is fine as-is.